test/factory.js

```diff
@@ -52,6 +64,22 @@ function Factory (lab, data, udaruCore) {
     })
   }
 
+  function createSharedPolicies (done) {
+    if (!data.sharedPolicies) return done()
+
+    async.mapValues(data.sharedPolicies, (policy, key, next) => {
+      udaru.policies.createShared(Object.assign({}, DEFAULT_SHARED_POLICY, _.pick(policy, 'id', 'name', 'version', 'statements')), (err, res) => {
+        if (err) return next(err)
+        next(null, res)
+      })
```

Comment:
can this:
```
(err, res) => {
 +        if (err) return next(err)
 +        next(null, res)
 +      }
```
be simplified to just passing `next`?

Reply:
In theory, since this is the predicate of the map we want to make sure we are "returning" only `res` (whatever the original function gets in input) but in this situation I think it's indeed safe to simplify it